ResearchUXFactory/NSDate+ISO8601Format.m

```diff
@@ -72,6 +72,34 @@ + (NSDateFormatter *)ISO8601DateOnlyformatter
     return formatter;
 }
 
++ (NSDateFormatter *)UnitedStatesDateOnlyformatter
```

Comment:
And yes, its only intended for use in the `dateValue` and `timeValue` implementations. Really strings *should* come from the server as ISO8601 and rules should be validated there, but this will work-around that issue until it's fixed and will be reverse-compatible even then.